components/up-job.libsonnet

```diff
@@ -128,6 +129,7 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
                 |||
                   curl --request POST \
                       --silent \
+                      %s \
```

Comment:
When `u.config.oidc` isn't set, it will generate a command like:

```console
$ curl --request POST \
--silent \
 \
--url https://dex.dex.svc.cluster.local:5556/dex/token \
--header 'content-type: application/x-www-form-urlencoded' \
--data grant_type=password \
--data username=admin@example.com \
--data password=password \
--data client_id=test \
--data client_secret=ZXhhbXBsZS1hcHAtc2VjcmV0 \
--data scope="openid email" | sed 's/^{.*"id_token":[^"]*"\([^"]*\)".*}/\1/'
```

I confirmed that this works (at least with my curl).